### ISSUE TITLE
Uploading to ITS is broken!

### ISSUE THREAD

**Issue by tritter** [firebase/firebase-ios-sdk#7936]
### Describe your environment

  * Xcode version: 12.4 (12D4e)
  * Firebase SDK version: 7.11.0
  * Installation method: `Swift Package Manager`
  * Firebase Component: Core
  * iOS Target & SDK(14.1)
### Step 2: Describe the problem
I can't upload an app to Apple. After processing I get an E-Mail saying:

```
Please correct the following issues, then upload again.

ITMS-90725: SDK Version Issue - This app was built with the iOS SDK. All iOS apps submitted to the App Store must be built with the iOS 13 SDK or later, included in Xcode 11 or later. Further, starting April 26, 2021 iOS apps submitted to the App Store must be built with the iOS 14 SDK or later, included in Xcode 12 or later.

```

I think Apple changed their minimum target? Without FirebaseCore I'm able to process the build!

#### Steps to reproduce:

What happened? How can we make the problem occur?
Empty Project, add via SMP, upload via Xcode to AppStore Connect

If you have a downloadable sample project that reproduces the bug you're reporting, you will
likely receive a faster response on your issue.

**Comment by ryanwilson** [comment]
Hi @tritter, sorry for the troubles. Can you share which products you're including via SwiftPM? Everything except Analytics is distributed as source so sources should be built using the Xcode version you're using. Note that there's a separate issue with binary SDKs (#6472) that is worth looking at as well, but this is a separate error you're seeing.

Can you also confirm you're not using the Xcode 12.5 RC released yesterday? I've seen similar messages using betas or RCs before, but you did call out Xcode 12.4 so it's likely not that.

Thanks!

**Comment by tritter** [comment]
@ryanwilson No problem, happy to help. I Only included the default via SPM. And yes I'm aware of that issue! I solved it by adding one of the scripts which is mentioned there, which let me upload the IPA last time. So now I just changed something in my code and tried to rebuild, without success, so also with 7.10.0 it worked before. Seems like Apple toggled some switches again on their side... Last upload was the 16th April.

![Screenshot 2021-04-22 at 16 22 53](https://user-images.githubusercontent.com/1229109/115731340-582a1f00-a387-11eb-959f-716aa3fab6c4.png)

Script I used after the archive step:
`# Type a script or drag a script file from your workspace to insert its path.
# https://github.com/firebase/firebase-ios-sdk/issues/6472
LOGFILE="${ARCHIVE_PATH}/static-frameworks.log"
echo "Removing static frameworks from ${WRAPPER_NAME} archive" > $LOGFILE
find "${ARCHIVE_PRODUCTS_PATH}/Applications/${WRAPPER_NAME}" -name '*.framework' -print0 | while IFS= read -r -d '' fm; do
    name=$(basename "${fm}" .framework)
    target="${fm}/${name}"
    echo "Checking: ${fm}" >> $LOGFILE
    if file "${target}" | grep -q "current ar archive"; then
        rm -rf "${fm}"
        echo "Removed static framework: ${fm}" >> $LOGFILE
    fi
done
`

**Comment by paulb777** [comment]
Core is not a product available via Swift Package Manager. See https://github.com/firebase/firebase-ios-sdk/blob/master/Package.swift#L25 for the list.  

Which Firebase products are linked into the app?

**Comment by tritter** [comment]
You're completely right. To the target it it is only "FirebaseAnalytics" Thanks

**Comment by tritter** [comment]
Hi @paulb777,

Seems to be fixed with the RC of Xcode 12.5!
Just submitted successfully just by upgrading to Xcode 12.5, I also removed the archive step now. So https://github.com/firebase/firebase-ios-sdk/issues/6472 should be solved too with 12.5

Thanks for responding so quickly,
Thom
